ophyd/areadetector/filestore_mixins.py

```diff
@@ -169,6 +168,43 @@ def stage(self):
         self._resource = fs.insert_resource('AD_TIFF', self._fp, res_kwargs)
 
 
+class FileStoreTIFFSquashing(FileStoreBase):
```

Comment:
Would you throw a basic example of how to assemble this into the docstring? Maybe an excerpt from your code snippet in the conversation here.